TypeScript error when passing enzymeToJson to setSerializers

Hello, here is my setup code:

**test-setup.ts**
```ts
import * as snapshotDiff from 'snapshot-diff';

// This is the same function exposed in `enzyme-to-json/serializer' as per README
import { createSerializer } from 'enzyme-to-json'; 

const enzymeToJsonSerializer = createSerializer();
snapshotDiff.setSerializers([enzymeToJsonSerializer]);
```

This produces the following error:

```bash
error TS2741: Property 'diffOptions' is missing in type 'JestSerializer' but required in type 'Serializer'.
```

This seems like a mismatch between the [`Serializer`](https://github.com/jest-community/snapshot-diff/blob/652ce486793f874313541b9af9923c16503ddbfa/index.d.ts#L26-L30) type in this repository and the [`JestSerializer`](https://github.com/adriantoine/enzyme-to-json/blob/master/index.d.ts#L22-L25) type from `enzyme-to-json`.

My understanding is that a serializer should only have two required properties (`print` and `test`) so perhaps `diffOptions` needs to be optional? Note that it appears the source code expects [it to be optional already](https://github.com/jest-community/snapshot-diff/blob/652ce486793f874313541b9af9923c16503ddbfa/src/index.js#L43-L45)

If I try to make that change I get:

```bash
 error TS2322: Type 'JestSerializer' is not assignable to type 'Serializer'.
  Types of property 'print' are incompatible.
    Type '(CommonWrapper: CommonWrapper<{}, {}, Component<{}, {}, any>>, serializer: JestSerializer) => Json' is not assignable to type '(value: any, _serializer?: Function | undefined) => string'.
      Types of parameters 'serializer' and '_serializer' are incompatible.
        Type 'Function | undefined' is not assignable to type 'JestSerializer'.
          Type 'undefined' is not assignable to type 'JestSerializer'.
```

Let me know if I can provide further debugging help, thanks

--- a/index.d.ts
+++ b/index.d.ts
@@ -25,8 +25,8 @@ declare namespace jest {
 
 interface Serializer {
   test: (value: any) => boolean;
-  print: (value: any, _serializer?: Function) => string;
-  diffOptions: (valueA: any, valueB: any) => DiffOptions;
+  print: (value: any, _serializer?: any) => any;
+  diffOptions?: (valueA: any, valueB: any) => DiffOptions;
 }
 
 declare module 'snapshot-diff' {

